seatunnel-e2e/seatunnel-flink-e2e/src/test/resources/assertion/fakesource_to_assert.conf

```diff
@@ -0,0 +1,88 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  # You can set flink configuration here
+  execution.parallelism = 1
+  #execution.checkpoint.interval = 10000
+  #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
+}
+
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+    FakeSource {
+      result_table_name = "fake"
+      field_name = "name,age"
+    }
+
+  # If you would like to get more information about how to configure seatunnel and see full list of source plugins,
+  # please go to https://seatunnel.apache.org/docs/flink/configuration/source-plugins/Fake
+}
+
+transform {
+    sql {
+      sql = "select name,age from fake"
+    }
+
+  # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+  # please go to https://seatunnel.apache.org/docs/flink/configuration/transform-plugins/Sql
+}
+
+sink {
+   AssertSink {
+       rules = 
+           [{
+               field_name = name
+               field_type = string
+               field_value = [
+                   {
+                       rule_type = NOT_NULL
+                   },
+                   {
+                       rule_type = MIN_LENGTH
+                       rule_value = 3
+                   },
+                   {
+                        rule_type = MAX_LENGTH
+                        rule_value = 5
+                   }
+               ]
+           },{
+               field_name = age
+               field_type = int
+               field_value = [
+                   {
+                       rule_type = NOT_NULL
+                   },
+                   {
+                       rule_type = MIN
+                       rule_value = 10
+                   },
+                   {
+                        rule_type = MAX
+                        rule_value = 20
+                   }
+               ]
+           }
```

Comment:
You may need to modify this length, I see the default mock name of FakeSource is
```
"Gary", "Ricky Huo", "Kid Xiong"
```
and the age is between 1-100.

Or you need to set a mock schema.

Reply:
Can you tell me where can i see the IT error?

Reply:
The error is in container, and it doesn't display, this is strange, but you can test on your environment.

Reply:
Fine. I will amend the assert config first.